content/2016-05-14-pydelhi-meetup.md

```diff
@@ -0,0 +1,77 @@
+Title: PyDelhi MayPy
+Slug: pydelhi-14-05-16
+Date: 2016-01-17 12:00
+Category: Meetup
+Author: Harshit Tyagi
+Email: harshit.bvcoe@gmail.com
+
+![Think open-source]({filename}/images/pydelhi-14-05-16.jpg)
```

Comment:
This is one of those photos used below just that I forgot to change the alt tag from Think open-source to PyDelhi.